levels.go

```diff
@@ -753,24 +753,46 @@ func hasAnyPrefixes(s []byte, listOfPrefixes [][]byte) bool {
 	return false
 }
 
-func containsPrefix(smallValue, largeValue, prefix []byte) bool {
+func containsPrefix(table *table.Table, prefix []byte) bool {
+	smallValue := table.Smallest()
+	largeValue := table.Biggest()
 	if bytes.HasPrefix(smallValue, prefix) {
 		return true
 	}
 	if bytes.HasPrefix(largeValue, prefix) {
 		return true
 	}
+	isPresent := func() bool {
+		ti := table.NewIterator(false)
+		defer ti.Close()
+		// In table iterator's Seek, we assume that key has timestamp in last 8 bytes. We set ts=0,
+		// so that we don't skip the key prefixed with prefix.
+		prefixWithTs := make([]byte, len(prefix)+8)
+		copy(prefixWithTs[:len(prefix)], prefix)
+		prefixWithTs = prefixWithTs[:cap(prefixWithTs)]
```

Comment:
Maybe you don't need this. Also, the cap could be much bigger than the actual length.

Reply:
Done.